.github/CODEOWNERS

```diff
@@ -612,6 +612,7 @@ app/sidekiq/cypress_viewport_updater @department-of-veterans-affairs/backend-rev
 app/sidekiq/decision_review @department-of-veterans-affairs/benefits-decision-reviews-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/sidekiq/decision_review/form4142_submit.rb @department-of-veterans-affairs/benefits-decision-reviews-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/sidekiq/decision_review/submit_upload.rb @department-of-veterans-affairs/benefits-decision-reviews-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+app/sidekiq/delete_old_pii_logs_job.rb @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
```

Comment:
This was created by the 1010 team and they do call `PersonalInformationLog` a lot... but it's also called elsewhere. Do you think they would own it or do you think it should be owned by Platform?

Reply:
I vote we leave as is, but mark as ophaned

Reply:
ok, sounds good. I might be the only one looking at this board, but I've marked it as orphaned [here](https://github.com/orgs/department-of-veterans-affairs/projects/1166/views/1?filterQuery=deleteold&pane=issue&itemId=50674286) 😄